client1.env

```diff
@@ -0,0 +1,13 @@
+# source info for client1
+
+APICIP=127.0.0.1
+APICUSER=admin
+
+# use base64 to encode string to prevent eyeball theft
+APICPASS64="cGFzc3dvcmQ="
+
+ACT="topology/pod-1/node-105/sys/aggr-[po9]"
+SBY="topology/pod-1/node-106/sys/aggr-[po6]"
+
+# callback must be absolute path
+CALLBACK=/home/admin/active_standby/client1_callback.sh
```

Comment:
I didn't highlight it well in the README document, but the script actually has built in support for env variables (so you do not need to pass them as cli parameters)

Basically, prefix all env variables with AS and then slugify the cli parameter

For example
```
AS_APIC_ADDRESS=127.0.0.1  
AS_APIC_USER=admin 
AS_PC_ACTIVE="topology/pod-1/node-105/sys/aggr-[po9]"
AS_PC_STANDBY="topology/pod-1/node-106/sys/aggr-[po6]"
AS_CALLBACK=/home/admin/active_standby/client1_callback.sh
```
Then you can just call the script with

`python2.7 active_standby.py --apic-password=${APICPASS}`